### PR TITLE
refactor about page to include club details

### PR DESCRIPTION
**Pull request by akshtshrma24**

### CODE REVIEW

**Review by danknessdra — APPROVED** [review]
LGTM

File: src/Pages/About/About.js [hunk 1]
```diff
@@ -1,37 +1,99 @@
...
+                    <CardText>
+                      Set yourself apart from others by learning skills and resume help
+                    </CardText>
+                  </CardBody>
+                </Card>
+              </Col>
+            </Row>
+            <Row className='center'>
+              <Col className='text-col'>
+                <h1>learn while having fun, making friends and eating pancakes</h1>
```
Comment (Luceium) [inline src/Pages/About/About.js:54]: Capitalize the "l" in learning to be consistent

File: src/Pages/About/About.js [hunk 2]
```diff
@@ -1,37 +1,99 @@
...
-            <img className='img-size' src={photo} alt="scephoto" />
-          ))}
-        </div>
+        <Container>
+          <Row className='center about-text'>
+            <Row className='center'>
+              <Col className='text-col'>
+                <h1> The Next Frontier of Innovation at San Jose State.</h1>
+                <p>
+                  The Software and Computer Engineering Society aims
```
Comment (Luceium) [inline src/Pages/About/About.js:23]: The p tags under the three headings each seem to have a different font size. Also, the p tag on line 22 isn't bolded like the other two.

File: src/Pages/About/About.js [hunk 3]
```diff
@@ -1,37 +1,99 @@
...
+                  <CardBody>
+                    <CardText>
+                      Visit company campuses, attend company info sessions and club events
+                    </CardText>
+                  </CardBody>
+                </Card>
+                <Card style={{ minWidth: '40vw' }}>
+                  <CardBody>
+                    <CardText>
+                      Set yourself apart from others by learning skills and resume help
```
Comment (Luceium) [inline src/Pages/About/About.js:46]: I feel like there's a grammar issue with "and resume help" maybe say, "receiving resume support"

**Review by Luceium — APPROVED** [review]
Left some comments on style. Other than that lgtm

**Pull request merged**